trigger/process.go

```diff
@@ -96,7 +97,13 @@ type ProcessorArgs struct {
 	EventID                      string
 	DefinitionID                 string
 	Alias                        string
+	ProjectID                    string
 	UnscheduleDownstreamVersions bool
+	PushInfo                     PushInfo
+}
+
+type PushInfo struct {
```

Comment:
Should comment on this as a public struct (and/or make private); would be curious to know why we went with a struct her also

Reply:
Ah, during testing I believe I was using more fields here. I'll just remove this struct entirely.